### DISCUSSION TITLE
키글챌 22주차 반성문

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#241] [6. 반성문]
안녕하세요 여러분.. 첫 글을 반성문으로서 인사 드립니다.
오늘 이제 막 프리코스 4주차 마지막과제 제출하고 소감문을 올리고 오는 길입니닷.. 일주일이 실험에 프리코스에 너무 정신없이 돌아가서 글을 주에 하나 쓰는 것 마저도 계속 잊고 까먹고 ㅠㅠ 이렇게 되었네욥.. _(프리코스 관련 글이라도 쓸걸😭)_

이제 프리코스도 끝나서 이번주부터는 스터디 하면서 MVC패턴이랑 jpa공부(조금씩..) 매일 하는 것이 목표인데 여기도 글을 열심히 써서 올려보겠습니닷.. 기록하는 습관!!

**Comment by shkisme** [comment]
화이팅~~~~!
